### DISCUSSION TITLE
Market-Ladys and problems to deliver higher goods

### DISCUSSION THREAD

**Discussion by Italicus66** [Keriew/augustus#813] [General]
Ave Romans,
I have some problems to hold medium housing-levels of Insulae. Especially the limit between Large Casa and Small Insulae make it painful visible in my city.
It seems to me that the Market-Ladys have some problems to deliver medium goods like pottery, furniture and oil, so my Insulae will **inescapable** collapsing back to a lower level.
I guess it cannot be intentionally by the game?

For showing you the problem, I will give an example and attache a city-map of mine:
_I do play always with the latest game-build, and I hope with this map my individual game-settings will saved,too._
There is nothing to do just wait maybe 5 minutes. _(The City should be stable except this special problem I tell here)_
You will see that there should be enough Markets (look at special maps), and a good accessibility to a lot of Warehouses with the goods pottery, furniture and oil are given, too. But all this cannot avoid, that the Insulae will collapsing after a short time, because there is no pottery, furniture and oil in the houses.
What would you say?

By the way, big thanks to the Augustus-Team! I do enjoy the new features of C3, and I'm very curious for your new ideas!
:-)
[Londinium_Italicus-66-001.svx.zip](https://github.com/Keriew/augustus/files/10566448/Londinium_Italicus-66-001.svx.zip)

**Comment by crudelios** [comment]
This is a fairly common recurrence.

Markets will prioritise food, with wheat being first on the list.

The more markets you have, the more strain you'll have on the wheat production, as all of them go fetch whatever wheat is available.

The solution is to specialise the markets. Get some to only fetch wheat and others to fetch goods. That will solve your issue almost instantly.

**Reply by Italicus66** [reply]
...it works perfect, thank you again!  :-D

**Reply by capsicumw** [reply]
With release 3.2 I have seen the pottery issue at any time the city needs workers. The local area of a city may have many workers but the market lady will refuse to get pottery until every job in the city is filled. (or at least every industry job.) 
Even with a fully staffed warehouse and granery one tile away from the market (both near full capacity and set to using and trading; with mix of wheat and other food).

**Comment by Italicus66** [comment]
Aahh oki, I didn't know that. Thank you, dear crudelios... :-)
Does it mean, that we will see one day special markers on the markets, so one can see in one view, what is tradet there? ...Maybe flags like for warehouses?

**Reply by crudelios** [reply]
Humm, a visual cue...? I'd like that, but I don't know what would look good in the little space the market uses.

**Reply by Italicus66** [reply]
I just tried it... One could think about, if it would be a good look. The example pictures in the zip-file have the same width and height as the original ones from the C3.sg2 file. The point is one would get an enormous quantity of combinations for each possible market setting.. Hmm...? :-D
[examples of trade settings for markets.zip](https://github.com/Keriew/augustus/files/10577759/examples.of.trade.settings.for.markets.zip)

**Reply by crudelios** [reply]
That is a bit messy though, especially considering we're going to add new goods and food soon enough.

**Reply by maoxp09** [reply]
Hello, do you have a submission email? I have created some new materials, how can I submit them?
